package.json

```diff
@@ -70,8 +70,8 @@
     "docker:down": "docker compose down -v"
   },
   "dependencies": {
+    "@irys/arweave": "^0.0.3",
```

Comment:
hm - what are they providing here?

Reply:
they provide an ESM index file that will pass the Web compatible Arweave to an ESM React project